source/help/settings/account-settings.rst

```diff
@@ -215,10 +219,17 @@ Select what language Mattermost displays in the user interface. Options include:
 - 中文 (繁體) - Traditional Chinese
 - 日本語 - Japanese
 
+Sidebar
+--------
+
+Automatically Close Direct Messages
```

Comment:
Propose mentioning this is experimental and needs to be enabled by the System Admin first

Reply:
Done, thanks